src/Log/SentryLogger.php

```diff
@@ -332,4 +333,80 @@ public static function backtrace(array $record) : array
         ]);
     }
 
+    /**
+     * Get the information about the current release, if possible
+     * @return array
+     */
+    protected function getGitInfo()
+    {
+        // Initialise empty array to return
+        $return = [];
+        // If the file operations error out, we need to catch it
+        try {
+            $return = $this->getGitCommitMessage($return);
+        } catch (\Exception $exception) {
+            // Default message. As it's an extra, it's not overly crowding the interface
+            $return['Message'] = 'No git repo found or inaccessible';
+        }
+
+        return $return;
+    }
+
+    /**
+     * Get the HEAD tag or the release
+     *
+     * @return string
+     */
+    public static function getGitHead()
+    {
+        $head = file_exists(Director::baseFolder() . '/.git/HEAD');
+        if ($head) {
+            $head = explode(':', file_get_contents(Director::baseFolder() . '/.git/HEAD'));
+            $ref = file_get_contents(Director::baseFolder() . '/.git/' . trim($head[1]));
+            $ref = self::getGitTag($ref);
+            return $ref;
+        }
+
+        return '';
+    }
+
+    /**
+     * Check if we are on a tagged release and if so, use that to set the client release
+     * @param $ref
+     * @return mixed
+     */
+    protected static function getGitTag($ref)
+    {
+        $data = file_get_contents(Director::baseFolder() . './git/packed-refs');
+        if ($data) {
+            $data = explode('\n', $data);
```

Comment:
This doesn't do what you think it does. You need either `"\n"` or even better `PHP_EOL` to serve Windows envs.